app/src/gui/components/notebook/record_table.tsx

```diff
@@ -175,6 +180,9 @@ function getDataForColumn({
       case 'KIND':
         return uiSpecification.viewsets[record.type]?.label ?? record.type;
 
+      case 'SYNC_STATUS':
+        return record.synced ? '✅' : '❌';
```

Comment:
Should there be a status which is 'unchecked' - seeing a bright red X on all new records until the 2 minute timer fires is a questionable user experience. Red means error to many users. What about you show 'Syncing...' or a gray line or something until the check happens?

Reply:
Replaced with a nice green cloud icon and an amber 'pending' icon.